test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Nice job so far setting up the `test.md` file, but it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) that this file should be ignored by git😉 which means that it should be added to your `.gitignore` file so as to avoid it from being uploaded to github🤗🤗

<img width="572" alt="test_imp2" src="https://user-images.githubusercontent.com/64643995/219360140-2ebf3953-d02f-4e11-a82d-081739780824.png">